tests/gql_behave/tests/memgraph_V1/features/case.feature

```diff
@@ -57,3 +57,33 @@ Feature: Case
         Then the result should be:
             |           z                 |
             | ['nottwo', 'two', 'nottwo'] |
+
+    Scenario: Simple CASE nullcheck does not have match:
+        Given an empty graph
+        When executing query:
+            """
+            WITH 2 AS name RETURN CASE name WHEN 3 THEN 'works' WHEN null THEN "doesn't work" ELSE 'something went wrong' END
+            """
+        Then the result should be:
+            |  CASE name WHEN 3 THEN 'works' WHEN null THEN "doesn't work" ELSE 'something went wrong' END |
+            |  'something went wrong'                                                                      |
```

Comment:
```suggestion
            WITH 2 AS name RETURN CASE name WHEN 3 THEN 'something went wrong' WHEN null THEN "doesn't work" ELSE 'works' END
            """
        Then the result should be:
            |  CASE name WHEN 3 THEN 'something went wrong' WHEN null THEN "doesn't work" ELSE 'works' END |
            |  'works'                                                                      |
```

Seeing the expected output it is a bit confusing to see `something went wrong` as expected result, that's why I proposed these changes.